service/src/main/java/org/sunbird/dao/otp/impl/OTPDaoImpl.java

```diff
@@ -90,4 +90,89 @@ public void updateAttemptCount(Map<String, Object> otpDetails, RequestContext co
     cassandraOperation.updateRecordWithTTL(
         JsonKey.SUNBIRD, TABLE_NAME, request, compositeKey, ttl, context);
   }
+
+
+  /**
+   * Inserts OTP details into the database.
+   *
+   * @param type              The type of the OTP (e.g., email, phone).
+   * @param key               The key associated with the OTP.
+   * @param otp               The OTP to be inserted.
+   * @param contextType       The type of context associated with the OTP.
+   * @param contextAttributes The attributes of the context associated with the OTP.
+   * @param context           The request context.
+   */
+  @Override
+  public void insertOTPDetailsV3(String type, String key, String otp, String contextType, String contextAttributes, RequestContext context) {
+    // Create a map to store OTP details.
+    Map<String, Object> request = new HashMap<>();
+    // Populate the map with OTP details.
+    request.put(JsonKey.TYPE, type);
+    request.put(JsonKey.KEY, key);
+    request.put(JsonKey.OTP, otp);
+    request.put(JsonKey.CONTEXT_TYPE, contextType);
+    request.put(JsonKey.CONTEXT_ATTRIBUTES, contextAttributes);
```

Comment:
Null check is required

Reply:
1. Null check is handled in the OTPActor class,if it is null empty string is recieved.